packages/foreman/rubygem-hammer_cli_foreman/rubygem-hammer_cli_foreman.spec

```diff
@@ -4,19 +4,19 @@
 %global gem_name hammer_cli_foreman
 %global confdir hammer
 
-%global release 2
-%global prerelease .pre.develop
+%global release 3
+%global prerelease pre.develop
 
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
 Summary: Universal command-line interface for Foreman
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 0.15
```

Comment:
Current git master is also 0.16 which we should match: https://github.com/theforeman/hammer-cli-foreman/commit/fd102084e3a562e1e437568412b2de3c3f51561e